x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/pipeline_select.tsx

```diff
@@ -59,14 +62,20 @@ export const PipelineSelect: React.FC = () => {
       listProps={{
         activeOptionIndex: getActiveOptionIndex(),
         bordered: true,
-        rowHeight: useIsWithinMaxBreakpoint('s') ? 120 : 90,
         showIcons: true,
         onFocusBadge: false,
+        rowHeight,
+      }}
+      searchProps={{
```

Comment:
Is there a way for us to know the amount of matchingOptions as a variable within this component? Then we'd be able to just calculate the height directly in this component instead of having to hook into an onChange

Reply:
Not as far as I know. I used an approach provided by @tkajtoch from the EUI team. @tkajtoch, can you check this implementation?

Reply:
You could set `isPreFiltered: true` and filter `options` outside `<EuiSelectable>`, but it's more work, and you'd still need to use `searchProps.onChange` to get input value changes. I think your current implementation is on point!

If this pattern starts to appear in more places, we may consider adding a new prop to `<EuiSelectable>` to control this behavior directly in EUI.

@Mikep86 was there a specific design requirement to update the height dynamically in this view? If so, who was the designer?

Reply:
@tkajtoch It's not really a design requirement, more like a request from @demjened to set a max height that the list can use. I initially set the `height` prop to a static value to implement this, but then `EuiSelectable` rendered a bunch of empty space when `height` > `<option_count> * rowHeight`. That lead me to the approach of setting the height dynamically.

A dedicated prop for controlling this behavior would be greatly appreciated 😃